src/main/java/core/basesyntax/operations/ReturnOperation.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.operations;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public class ReturnOperation implements OperationHandler {
+    private final Storage storage;
+
+    public ReturnOperation(Storage storage) {
+        this.storage = storage;
+    }
+
+    @Override
+    public void apply(FruitTransaction transaction) {
+        if (storage == null) {

```

Comment:
The check for `storage == null` is unnecessary here since it's being injected through the constructor. If you want to enforce non-null storage, consider adding the check in the constructor instead.

Reply:
fix comment